sagemaker-featurestore/feature_store_introduction.ipynb

```diff
@@ -473,6 +473,152 @@
     "all_records"
```

Comment:
<p>Formatting nit: backticks around <code>customers_feature_group</code>?  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3515/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
<p>added <code></code> around customers_feature_group in new version.</p>